test/library/packages/Sort/RadixSort/radixMSBModule.chpl

```diff
@@ -0,0 +1,83 @@
+ use BitOps;
```

Comment:
If there is anything outside of a module scope in a Chapel program, the compiler will create an implicit module based on the file name. In this case, we are implicitly creating the following module hierarchy:

```chpl
module radixMSBModule {
  use BitOps;
  use Sort;
  use Random;
  config param bucketBits=8;

  module MSBmodule {
    ...
  }
}
```

You will want to put the `use` and `config param` statements inside the `MSBmodule` scope below to avoid this implicit module from being created.